### PR TITLE
useUpdateBallots hook

### LINKED ISSUE #196: Backend: Create a useUpdateBallots hook
## **User story** 
**Who:** As a developer
**What:** I want to create a useUpdateBallots hook.
**Why:** So that useUpdateBallots hook will only fetch and update rank of candidates.

### PR DESCRIPTION
**Pull request by Rasim-Deniz**
# Description

A useUpdateBallots hook that has only one work, which is fetching and updating rank of candidates, is created.

Fixes #196 

# How to test?

Call the useHook function with regarding parameters.

# Checklist

- [X] I have performed a self-review of my own code
- [X] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [X] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [X] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [X] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/containers/useUpdateBallots.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      );
+      if (!response.ok) {
+        setError(
+          `Error updating ballot: ${response.status}. ${response.statusText}`,
+        );
+      }
+    })();
+  }, [userId, boardId, candidatesBody]);
+
+  return { error };
```
Comment (Rasim-Deniz) [inline src/client/containers/useUpdateBallots.js:27]: Same problem here. What should the dependency array hold here and is this return correct as it only returns error?

Comment (senner007) [reply src/client/containers/useUpdateBallots.js:27]: You don't need a useEffect here. Just return a method to update the ballots. The method will take the current array of candidate sortings

**Review by senner007 — CHANGES_REQUESTED** [review]

**Comment by senner007** [comment]
@Rasim-Deniz 
I think you should collect the useCandidates and useBallots and CandidateList in one pr. Then its easier for us to test and get it working

**Comment by Rasim-Deniz** [comment]
> @Rasim-Deniz 
> I think you should collect the useCandidates and useBallots and CandidateList in one pr. Then its easier for us to test and get it working 

Yes, you are absolutely right. Should I delete the PRs that I created for useCandidates and useBallots?

**Comment by senner007** [comment]
> > @Rasim-Deniz
> > I think you should collect the useCandidates and useBallots and CandidateList in one pr. Then its easier for us to test and get it working
> 
> Yes, you are absolutely right. Should I delete the PRs that I created for useCandidates and useBallots?

Yes I think so. I'll be out but available tomorrow

**Pull request closed without merging**